system-addon/lib/TopSitesFeed.jsm

```diff
@@ -73,6 +77,10 @@ this.TopSitesFeed = class TopSitesFeed {
           this.refresh(action);
         }
         break;
+      case at.OPEN_NEW_WINDOW:
+      case at.OPEN_PRIVATE_WINDOW:
+        this.openNewWindow(action);
```

Comment:
How about doing ``` openNewWindow(action, isPrivate = false)``` and then in the ```PRIVATE``` case call  ```this.openNewWindow(action, true);```. So we're covered both ways, and by default it just opens a regular window